feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -591,6 +595,9 @@ func testImplementDrain(ctx context.Context, t *testing.T, args *testArgs) {
 	p3Flow := createFlow(t, p3flow, args.top, &atePort3)
 	p4Flow := createFlow(t, p4flow, args.top, &atePort4)
 
+	args.ate.OTG().PushConfig(t, args.top)
+	args.ate.OTG().StartProtocols(t)
```

Comment:
Can add `otgutils.WaitForARP(t, args.ate.OTG(), args.top, "IPv4")` after this instead of waitOTGARPEntry.